src/Sentry/Infrastructure/DiagnosticId.cs

```diff
@@ -8,4 +8,13 @@ internal static class DiagnosticId
     /// </summary>
     internal const string ExperimentalFeature = "SENTRY0001";
 #endif
+
+    //TODO: QUESTION: Should we re-use the above for all [Experimental] features or have one ID per experimental feature?
+    internal const string ExperimentalSentryLogs = "SENTRY0002";
```

Comment:
I think it's OK to just have one... the main purpose here is to make sure people are aware/deliberate about opting into our experimental APIs. They'll get a separate warning in their code each time they use one/any of them. 

Potentially they could be disabling those in a directory.build.props file for their solution... in which case separate diagnostic ids would mean they'd need to disable warnings for our Heap Dumps feature and the Logs feature separately... seems like overkill.

Reply:
SGTM
I am thinking about adding a single `UrlFormat` string, used by both/all experimental features, linking to a dedicated "Experimental Features" page in our _docs_. But will follow up with that in a follow-up PR.